lib/rules/sort-imports.js

```diff
@@ -43,12 +45,81 @@ module.exports = {
     },
 
     create: function(context) {
-
         const configuration = context.options[0] || {},
             ignoreCase = configuration.ignoreCase || false,
             ignoreMemberSort = configuration.ignoreMemberSort || false,
-            memberSyntaxSortOrder = configuration.memberSyntaxSortOrder || ["none", "all", "multiple", "single"];
-        let previousDeclaration = null;
+            memberSyntaxSortOrder = configuration.memberSyntaxSortOrder || ["none", "all", "multiple", "single"],
+            sourceCode = context.getSourceCode(),
+            reactRegExp = /\/?[^\/]*?react[^\/]*?$/ig;
```

Comment:
Sounds good, I mentioned in the description that the plan is to move it to options.